site/clustering-ssl.xml

```diff
@@ -57,12 +57,12 @@ cat server_certificate.pem server_key.pem &gt; rabbit.pem
 # /etc/rabbitmq/rabbitmq-env.conf and will preserve the existing
 # contents of that file if it already exists
 
-erl -noinput -eval 'io:format("ERL_SSL_PATH=~p~n",[code:lib_dir(ssl, ebin)]),halt().' > /tmp/ssl-path.txt
+erl -noinput -eval 'io:format("ERL_SSL_PATH=~s~n", [filename:dirname(code:which(inet_tls_dist))])' -s init stop > /tmp/ssl-path.txt
```

Comment:
I changed this after reading [this doc](https://www.erlang-solutions.com/blog/erlang-distribution-over-tls.html). Since the `inet_tls_dist` module is what actually does disterl, I think we should recommend this method.